packages/sanity/src/core/hooks/useClient.ts

```diff
@@ -26,9 +26,9 @@ export function useClient(clientOptions?: SourceClientOptions): SanityClient {
   const source = useSource()
   if (!clientOptions) {
     console.warn(
-      'Calling `useClient()` without specifying an API version is deprecated and will stop working in the next dev-preview release - please migrate to use `useClient({apiVersion: "2021-06-07"})`.',
+      'Calling `useClient()` without specifying an API version is deprecated and will stop working in the next major version - please specify a date, e.g. `useClient({apiVersion: "2025-02-10"})`.',
     )
-    return source.getClient({apiVersion: '2021-06-07'})
+    return source.getClient({apiVersion: 'v2025-02-07'})
```

Comment:
We soon want to bump the version used by all internal code, so I think this change is sensible, esp. considering we've had the above warning throughout the lifetime of v3

Reply:
Yes, will make a separate PR for it – would prefer a change like that to be subjected to a bit more testing :)